presto-druid/src/main/java/io/prestosql/plugin/druid/DruidConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.airlift.configuration.Config;
+
+public class DruidConfig
+{
+    private String sqlTimeZone = "America/Los_Angeles";
```

Comment:
Is this correct default? Maybe it should be taken from locale information? @findepi Any thoughts on this?

Reply:
What this value is used for?